src/edu/csus/ecs/pc2/ui/EditProblemSandboxPane.java

```diff
@@ -97,10 +94,13 @@ public class EditProblemSandboxPane extends JPanePlugin {
     
     //setting this to True will cause the Sandbox Pane GUI panels to show boundary borders... useful for layout debugging.
     private boolean debugShowPanelBorders = false;
+
+    // remember if we told the user this system does not support sandboxes so we don't keep nagging
+    private boolean noSandboxNagMessage = false;
     
     //setting this to True will override the prohibition on selecting a Sandbox when running on Windows.
     // Note that THIS IS FOR DEBUGGING PURPOSES; it does NOT imply any support for Windows sandboxing.
-    private boolean debugAllowSandboxSelectionOnWindows = true;
+    private boolean debugAllowSandboxSelectionOnWindows = false;
```

Comment:
I'm not convinced we should be disabling this boolean at this point in the development process.  I normally only have access to a Windows box for testing.  I believe that this change is going to stop me from testing this PR.

The original intent of this boolean (I was the one who added it to the code) was to _allow_ declaring a sandbox on a Windows machine for testing purposes (with the understanding that the developer/tester is smart enough, while testing sandbox issue, to know that it wasn't actually possible to _execute_ a sandbox on the windows test machine.

Then the idea was that, once ALL sandbox development and testing was completely, one of the very last things we'd do is THEN change this variable to `false` before releasing a production version of the code.

Reply:
Yes.  I had turned it off for testing.  I wonder, though, if in production we should turn it off (false)?  I'll put it back to do true since we are not making a production release at this time.

Reply:
Fixed in[ this commit](https://github.com/pc2ccs/pc2v9/pull/780/commits/c1c01fe246f2af6a8190f47e20cb9563add8b221).

Reply:
> Yes. I had turned it off for testing. I wonder, though, if in production we should turn it off (false)? ...

Yes, I fully agree we SHOULD NOT turn it "off" in production (that is, the variable should be set `false` in a production release) -- which I THINK is what you are saying (correct me if I'm wrong).    What we probably need is a single system-wide global flag like "isProductionRelease", and have all conditionals like this (there are other places in the code where similar things exist)  assign neither `true` nor `false` but instead the "global isProductionRelease" flag value.  That way, we have only a SINGLE place we have to remember to change when we finally "promote" a release to "production" (public).

Having said that, it's yet another one of those "Someday" things....